src/main-process/proxy/request-description.ts

```diff
@@ -0,0 +1,35 @@
+import { ProxyEntry } from '../../types/proxy-entry';
+
+import { LoadmillRequest } from './test-actions/transform';
+
+export const assignRequestDescriptions = (entries: ProxyEntry[], requests: LoadmillRequest[]): void => {
+  entries.forEach(entry => assignRequestDescription(entry, requests));
+};
+
+const assignRequestDescription = (entry: ProxyEntry, requests: LoadmillRequest[]): void => {
+  const request = requests.find(request => request.id === entry.id);
+  if (request) {
+    entry.request.description = request.description;
```

Comment:
`|| getDefaultRequestDescription` to be on the safe side if the request returns without description?